### PR TITLE
demo: API docs script adjustements

### LINKED ISSUE #927: API extraction script adjustements
2 issues spotted recently:
- [ ] - undocumented `@Input`s are showing up while they should not: https://ng-bootstrap.github.io/#/components/buttons
- [ ] - getters are documented properly, see: https://github.com/ng-bootstrap/ng-bootstrap/pull/903#discussion_r83921381

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Fixes #927

### CODE REVIEW

File: misc/api-doc.js [hunk 1]
```diff
@@ -13,21 +13,34 @@ const ANGULAR_LIFECYCLE_METHODS = [
...
+}
+
 function isInternalMember(member) {
   if (!member.symbol) {
     return true;
   }
 
   const jsDoc = ts.displayPartsToString(member.symbol.getDocumentationComment());
-  return jsDoc.trim().length === 0 || jsDoc.indexOf('@internal') > -1;
+  return jsDoc.indexOf('@internal') > -1;
```
Comment (wesleycho) [inline misc/api-doc.js]: If we mandate using Node 6, we could just do
```js
return jsDoc.includes('@internal');
```
Not a big deal either way

File: misc/api-doc.js [hunk 2]
```diff
@@ -13,21 +13,34 @@ const ANGULAR_LIFECYCLE_METHODS = [
...
 }
 
 function isAngularLifecycleHook(methodName) {
   return ANGULAR_LIFECYCLE_METHODS.indexOf(methodName) >= 0;
 }
 
+function isPrivate(member) {
+  return (member.flags & ts.NodeFlags.Private) !== 0;
```
Comment (wesleycho) [inline misc/api-doc.js]: What are the values possible for these items, booleans?

Comment (pkozlowski-opensource) [inline misc/api-doc.js]: Those are bit-flags. Not sure if there is any documentation for this besides TS source code. But for the record: I'm not changing this code, just moving it around.

Comment (wesleycho) [inline misc/api-doc.js]: I noticed that just afterwards, was just curious what data structures those were.

**Pull request closed without merging**